src/main/java/view/InputView.java

```diff
@@ -0,0 +1,26 @@
+package view;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public final class InputView {
+    private static final Scanner scanner = new Scanner(System.in);
+    public static final int START_POINT_INDEX = 0;
+    public static final int ARRIVAL_POINT_INDEX = 1;
+
+    public List<List<Integer>> readMovementRequest() {
+        System.out.println("출발점과 도착점의 위치를 알려주세요 ex.2,1 3,1");
+        final String input = scanner.nextLine();
+        final String[] splitInput = input.split(" ");
+        final List<Integer> startPoint = formatToIntegerList(splitInput, START_POINT_INDEX);
+        final List<Integer> arrivalPoint = formatToIntegerList(splitInput, ARRIVAL_POINT_INDEX);
+        return List.of(startPoint, arrivalPoint);
+    }
+
+    private static List<Integer> formatToIntegerList(String[] splitInput, int index) {
```

Comment:
```suggestion
    private static List<Integer> formatToIntegerList(final String[] splitInput, final int index) {
```
파라미터에 final을 붙이는 컨벤션을 정하신 것 같은데 빠진 부분을 전체적으로 확인 부탁드려요. ㅎㅎ

Reply:
[refactor: 빠진 final 키워드 추가](https://github.com/woowacourse/java-janggi/pull/5/commits/f7f9f3bc8b8d3919d6e36604411b7a9714821857) 에서 반영했습니다! ㅎㅎ